locations/hours.py

```diff
@@ -163,19 +163,19 @@
     "Söndag": "Su",
 }
 DAYS_SI = {
-    "po": "Mo",
```

Comment:
Please only add to these dictionaries. The existing usages were likely there for another spider that needed them.

Reply:
I actually intended for these dicts to be used with sanitise_day so we don't have to have multiple cases in there. My vote is for to update which spider added them.

Reply:
It was me that added the lowercase versions with hofer_si. Sorry everyone.

It's only used by a1_si, hofer_si and this. All work with the change

Reply:
@iandees It seems like all other language have first capital letter but it wasn't the case here which is why I changed them instead of duplicating them. If it works with the other spiders, I guess that we can mark it as resolved?